doc-Provisioning_Virtual_Machines_and_Hosts/topics/Provisioning_Requests.adoc

```diff
@@ -216,23 +234,42 @@ image:2323.png[]
 
+[NOTE]
+====
+ISO provisioning is performed using a Kicksart file. The customization template is required to provide Kickstart files for initial uploading of the operating system.
+====
+
+==== Cloud-Init
+
 .Cloud-Init Requirements
 * When creating a template using Red Hat Virtualization, install the *cloud-init* package on the source virtual machine. This enables Cloud-Init to source configuration scripts when a virtual machine built on that template boots.
 * See https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Virtualization/3.6/html-single/Virtual_Machine_Management_Guide/index.html#sect-Using_Cloud-Init_to_Automate_the_Configuration_of_Virtual_Machines[Using Cloud-Init to Automate the Configuration of Virtual Machines] in the Red Hat Virtualization _Administration Guide_ for more information on using Cloud-Init in a Red Hat Virtualization environment.
 * See the http://cloudinit.readthedocs.org/en/latest/[Cloud-Init Documentation] web site for example scripts.
 
+==== Kickstart
+ 
 .Kickstart Requirements for ISO Provisioning
-* The *Kickstart* file must be named *ks.cfg*.
-* Set the new virtual machine to power down after provisioning is complete.
-* {product-title_short} must use the virtual machine payload feature of Red Hat Virtualization to create a floppy disk containing the data from the selected customization template.
-* Customize the installer to include the data written to the floppy disk payload.
+* The *Kickstart* file in the root directory of the ISO must be named *ks.cfg*.
+* ISO Provisioning on Red Hat Virtualization will write the content generated by the customization template to the file named *ks.cfg* that will be attached to the virtual machine using the RHV floppy payload. If this content is to be included in the installation, the ISO image must be aware of this and include the contents of the file. This requires remastering of the ISO to include a custom Kickstart file. 
+* For more information, see link: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/installation_guide/sect-kickstart-howto[How Do You Perform a Kickstart Installation] in the Red Hat Enterprise Linux _Installation Guide_.
+* Set the new virtual machine to power down after provisioning is complete. Ensure that *Power on after creation* is unchecked on the Schedule tab when creating the VM provisioning dialog. 
 
-[example]
-.RHEL ISO with the following modifications:
-====
-* `isolinux.cfg` &ndash; add *ks=cdrom* to the *append* line
-* `ks.cfg` &ndash; which must minimally include:
+===== Sample ISO Including Kickstart File
 
+RHEL ISO with the following modifications:
+
+* Add a kickstart file named `ks.cfg` to the root of the ISO. See example below. 
+* Add `ks=cdrom:/ks.cfg` to the `append` line under the volume label in the `isolinux/isolinux.cfg` file to use the ISO as the installation method.   
```

Comment:
Thank you so much for this! It took me forever to figure out for the first time and now I was not able to recall what exectly I did those few months ago. I really cannot understand why this hasn't been there since the beginning.

Reply:
@jan-zmeskal no problem, it wasn't easy info to track down.